lib/queries/whereQuery.js

```diff
@@ -15,6 +15,7 @@ export function getFieldPlaceHolder(field, value) {
 
 export function getFieldType(field, searchableFields) {
     if (!searchableFields.length) {
+        console.warn('There are no allowed fields to be searched, all filters will be ignored');
```

Comment:
@ThieryMichel This cannot be a normal usage?
If this can be the case, there should not be a warning.